index.html

```diff
@@ -1,187 +1,284 @@
-                            <li class="tag-element">html</li>
-                        </ul>
-                    </div>
-                    <button type="button" class="see-project-btn">See Project</button>    
-                </div>
-                <div class="card card4">
-                    <div class="image-card-holder"></div>
-                    <div class="text-work-title">
-                        <p>Multi-Post Stories Gain+Glory</p>
-                    </div>
-                    <div class="card-tags">
-                        <ul>
-                            <li class="tag-element">Ruby on rails</li>
-                            <li class="tag-element">css</li>
-                            <li class="tag-element">JavScript</li>
-                            <li class="tag-element">html</li>
-                        </ul>
-                    </div>
-                    <button type="button" class="see-project-btn">See Project</button>    
-                </div>
-                <div class="card card5">
-                    <div class="image-card-holder"></div>
-                    <div class="text-work-title">
-                        <p>Multi-Post Stories Gain+Glory</p>
-                    </div>
-                    <div class="card-tags">
-                        <ul>
-                            <li class="tag-element">Ruby on rails</li>
-                            <li class="tag-element">css</li>
-                            <li class="tag-element">JavScript</li>
-                            <li class="tag-element">html</li>
-                        </ul>
-                    </div>
-                    <button type="button" class="see-project-btn">See Project</button>    
-                </div>
-                <div class="card card6">
-                    <div class="image-card-holder"></div>
-                    <div class="text-work-title">
-                        <p>Multi-Post Stories Gain+Glory</p>
-                    </div>
-                    <div class="card-tags">
-                        <ul>
-                            <li class="tag-element">Ruby on rails</li>
-                            <li class="tag-element">css</li>
-                            <li class="tag-element">JavScript</li>
-                            <li class="tag-element">html</li>
-                        </ul>
-                    </div>
-                    <button type="button" class="see-project-btn">See Project</button>    
-                </div>
-            </div>   
-        </section>
+          </div>
+        </div>
+      </section>
+
+      <section class="form-contact">
+        <h3 class="text-contact">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h3>
+        <form
+          id="template-form"
+          action="https://formspree.io/f/template-form"
+          method="post"
```

Comment:
- [ ] [REQUIRED] You have done a great job with the form implementation. However, the for is not submitting properly. 
This is because you separated the ID from the action.
Please implement your action like this and ensure your form submits successfully before requesting another review:
See that the `ID` is supposed to come up after the `action`.
`https://formspree.io/f/{form_id}`
#### The response currently from your form submission
![Screenshot from 2022-03-23 00-20-58](https://user-images.githubusercontent.com/58692590/159592835-f82bcdbf-ae4d-41ad-9bba-15b4b595952d.png)